tests/framework/extensions/provisioning/verify.go

```diff
@@ -70,7 +70,7 @@ func VerifyRKE1Cluster(t *testing.T, client *rancher.Client, clustersConfig *clu
 	require.NoError(t, err)
 	assert.NotEmpty(t, clusterToken)
 
-	err = nodestat.AllManagementNodeReady(client, cluster.ID)
+	err = nodestat.AllManagementNodeReady(client, cluster.ID, 30*time.Minute)
```

Comment:
For the time duration, change this to be a const. That way, it's easier in the future to change if needed.